src/main/resources/application.properties.EXAMPLE

```diff
@@ -30,6 +30,7 @@ metadb.cmo_request_update_topic=
 metadb.cmo_sample_update_topic=
 request_reply.patient_samples_topic=
 request_reply.crdb_mapping_topic=
+metadb.dmp_new_sample_topic=
```

Comment:
we should make a PR to the configuration repo to add this property to `cmo-metadb/application.properties` as well